packages/dev/inspector-v2/src/components/properties/materials/pbrBaseMaterialProperties.tsx

```diff
@@ -0,0 +1,317 @@
+                            }
+                        }}
+                    />
+                    <Collapse visible={bumpTexture !== null}>
+                        <BoundProperty component={SyncedSliderPropertyLine} label="Bump strength" target={bumpTexture!} propertyKey="level" min={0} max={2} step={0.01} />
+                    </Collapse>
+                    <BoundProperty component={SwitchPropertyLine} label="Use roughness from main texture" target={material.clearCoat} propertyKey="useRoughnessFromMainTexture" />
+                    <BoundProperty component={SwitchPropertyLine} label="Tint" target={material.clearCoat} propertyKey="isTintEnabled" />
+                    <Collapse visible={isTintEnabled}>
+                        <div>
+                            <BoundProperty component={Color3PropertyLine} label="Tint Color" target={material.clearCoat} propertyKey="tintColor" isLinearMode={true} />
+                            <BoundProperty
+                                component={SyncedSliderPropertyLine}
+                                label="At Distance"
+                                target={material.clearCoat}
+                                propertyKey="tintColorAtDistance"
+                                min={0}
+                                max={20}
+                                step={0.1}
+                            />
+                            <BoundProperty
+                                component={SyncedSliderPropertyLine}
+                                label="Tint Thickness"
+                                target={material.clearCoat}
+                                propertyKey="tintThickness"
+                                min={0}
+                                max={20}
+                                step={0.1}
+                            />
+                            <FileUploadLine
+                                label="Tint"
+                                accept=".jpg, .png, .tga, .dds, .env, .exr"
+                                onClick={(files) => {
+                                    if (files.length > 0) {
+                                        updateTexture(files[0], material.clearCoat.tintTexture);
+                                    }
+                                }}
+                            />
+                        </div>
+                    </Collapse>
+                </div>
+            </Collapse>
+        </>
+    );
+};
+
+export const PBRBaseMaterialIridescenceProperties: FunctionComponent<{ material: PBRBaseMaterial }> = (props) => {
+    const { material } = props;
+
+    const isEnabled = useProperty(material.iridescence, "isEnabled");
+
+    const updateTexture = (file: File, _texture: BaseTexture | null) => {
+        ReadFile(
+            file,
+            (data) => {
+                const blob = new Blob([data], { type: "octet/stream" });
+                const url = URL.createObjectURL(blob);

```

Comment:
Is this new code? Object urls are supposed to be released when we are done with them, otherwise they effectively leak memory. Do we have a pattern for this?